Mobile.FSharp.Templates.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>0.0.1</PackageVersion>
+    <PackageId>Mobile.FSharp.Templates</PackageId>
```

Comment:
I would name it `FSharp.Mobile.Templates` to follow the same convention than `FSharp.Android.Resource`

Reply:
But yes, I like the renaming. We can even add the missing old F# templates to the package.

Reply:
Thanks! I'll continue to work on templates in near time.

Reply:
Thanks! I'll merge this one for now